practice/flutter/flutter/models.py

```diff
@@ -0,0 +1,14 @@
+"""flutter Models."""
+
+from django.db import models
+
+
+class Flutt(models.Model):
+    """A flutt class model that stores text and a timestamp."""
+    text = models.TextField()
+    timestamp = models.DateTimeField(auto_now=True)
+
+    def __str__(self):
+        return self.text
+    def __repr__(self):
+        return 'Flutt(text={!r}, {!r})'.format(self.text, self.timestamp)
```

Comment:
Minor point: I recommend reprs to be executable Python code, so if this second argument had `timestamp=`, then it would work. Remember all Django model constructor calls have to use keyword argument passing.